deepspeech_pytorch/inference.py

```diff
@@ -69,13 +80,12 @@ def run_transcribe(audio_path: str,
                    model: DeepSpeech,
                    decoder: Decoder,
                    device: torch.device,
-                   use_half: bool):
+                   precision: int):
     spect = spect_parser.parse_audio(audio_path).contiguous()
     spect = spect.view(1, 1, spect.size(0), spect.size(1))
     spect = spect.to(device)
-    if use_half:
-        spect = spect.half()
     input_sizes = torch.IntTensor([spect.size(3)]).int()
-    out, output_sizes = model(spect, input_sizes)
+    with autocast(enabled=precision is 16):
```

Comment:
This will result in a `SyntaxWarning`. Ditto for `model.py` and `validation.py`

Suggested way was
```suggestion
    with autocast(enabled=precision == 16):
```